pokemongo_bot/inventory.py

```diff
@@ -106,6 +106,25 @@ def captured(self, pokemon_id):
             return False
         return self._data[pokemon_id]['times_captured'] > 0
 
+class Item(object):
```

Comment:
Yes someone did. Maybe for the best. But when merging conflicts I simply accepted my modified version. We need to heavily refactor this class anyway, so I didn’t take the time to move it again.